site/content/en/docs/user/stages-configuration.md

```diff
@@ -138,6 +139,37 @@ if a "Change Stage" is matched. The update action itself consequently causes a n
 However, this event-driven approach to applying Stages has a limitation: `kwok` won’t apply a Stage until a new event associated with that resource is received. To address the limitation,
 users can utilize the `immediateNextStage` field to make the controller apply Stages immediately rather than waiting for an event pushed from the apiserver.
 
+## How Delay is Calculated
+
+The delay time of applying a Stage is obtained by adding a constant time period and a randomized interval,
+which will be denoted by *duration* and *jitter* respectively in the following.
+
+Typically, *duration* is given by `durationMilliseconds` field, while *jitter* is calculated by *random*(`jitterDurationMilliseconds`-*duration*).
```

Comment:
Is it possible to make these two paras more clearly with bullets like:

```
- `durationMilliseconds`: provdes *duration*.
- `jitterDurationMilliseconds`: calculates *jitter* by *random*(`jitterDurationMilliseconds`-*duration*).
  It shall be larger than `durationMilliseconds` if you want to inject jitter to the delay.
  Otherwise, it will be used directly as the delay time without any randomization.

`kwok` also provides other fields to flexibly handle more situations. They are used to extract
and parse a RFC3339 timestamp field of a resource to help determine the delay time dynamically.

- `durationFrom`: calculates *duration* by `durationFrom`-*now*
- `jitterDurationFrom`: calculate *jitter* by *random*(`jitterDurationFrom`-*now*-*duration*),
  where *now* is the timestamp when the delay starts.

{{< hint "note" >}}
`durationFrom` and `jitterDurationFrom` have a higher priority than `durationMilliseconds` and `jitterDurationMilliseconds` if both are set.
{{< /hint >}}
```

Reply:
Hi, @windsonsea,
Thanks for the re-organization. This is much clearer!

But it seems that there is no "note" type for a hint (I also confirmed it locally).
reference https://hugo-book-demo.netlify.app/docs/shortcodes/hints/

Which one do you suggest to use here?

Reply:
How about `info`?

Reply:
Thanks!